### ISSUE TITLE
Domain/language specific common files folders

### ISSUE THREAD

**Issue by Zethson** [cookiejar/cookietemple#464]
I think that to scale it would be great if additionally to the common_files folder we had domain specific common_files folders such as e.g. python_common_files, which would be included if the domain matches.
Is imo required to scale.

Thoughts @Imipenem ?

**Comment by Imipenem** [comment]
Good Idea, but I really dont know if its worth the effort. I mean, let's take `python` as an example: What will get included into this folder?

**Comment by Zethson** [comment]
Not really sure as well.  Tuesday, 20 October 2020, 08:02PM +02:00 from Farwent  notifications@github.com :

**Comment by Imipenem** [comment]
I'd suggest closing this, but keep it in mind as cookietemple gets more templates. We may have to restructure some parts nevertheless so we can think about this when time has come

**Comment by Zethson** [comment]
Well, could aid with stuff that web-python and cli-python have in common.

So not just domain common, but also language common.

**Comment by Zethson** [comment]
Implemented in https://github.com/mlf-core/mlf-core and could be added to CT
